### ISSUE TITLE
Feedback UX multisig

### ISSUE THREAD

**Issue by malikfassi** [AstroquirksHQ/astroquirks-front#28]
in create multisig : 

- if one bubble is empty -> button continue disappear. it is confusing. we dont know why its not there. 

- empty bubble should not exist -> if empty, bubble should be auto deleted 

- clicking outside of add-key popup should close it. 

- cannot edit quorum

**Comment by malikfassi** [comment]
Empty state when moving from https://astroquirks.com/app to https://astroquirks.com/app/multisig is showing the floating astronaut. the app glitches when changing url.

**Comment by meriadec** [comment]
> Empty state when moving from https://astroquirks.com/app to https://astroquirks.com/app/multisig is showing the floating astronaut. the app glitches when changing url

indeed. For this one, I think it's inherent to Next.js static routing. Will try to dig a bit, else, we can also manage routing by ourselves instead of using next.js router
